content/home/examples/a-simple-component.md

```diff
@@ -1,9 +1,8 @@
 ---
-title: A Simple Component
+title: Un componente simple
 order: 0
 domid: hello-example
 ---
+Los componentes de React implementan un método `render()`, el cual toma los datos de entrada y regresa lo que se tiene que mostrar. En este ejemplo usaremos sintaxis tipo XML llamado JSX. Los datos de entrada que son pasados al componente pueden ser accesados en `render()` usando `this.props`.
```

Comment:
Saying "_toma **los** datos de entrada_" would be the equivalent to "_takes **the** input data_", and that's not what the original text says. We're talking about data in general not some data in specific to which we've referred to before.

Also, shouldn't `regresa` be `devuelve` o `retorna`? As far as I understand, `regresa` means to come back, which can be understood as returning but in a "returning from somewhere" sense, not a "returning something somewhere" sense.

The correct bit I suggest is `toma datos de entrada y devuelve lo que se tiene que mostrar`.

Reply:
1. `toma datos de entrada`, personally as a Spanish speaker it sounds a little bit odd, that's why I included the `los`, but I see the point.
2. `return` we also translate it to `regresa`, but I think you're right and it's better to use `devuelve`, because `retorna` is rarely used.
3. Sorry about `tipo` not professionally we use it also as a synonym to `similar`. But it'll be better to use `similar`
4. I'm used to see both `pueden ser` or `son` in translations of this case. But as you wish